workflow_step_execute.go

```diff
@@ -0,0 +1,85 @@
+package slack
+
+import (
+	"context"
+	"encoding/json"
+)
+
+type (
+	WorkflowStepCompletedRequest struct {
+		WorkflowStepExecuteID string             `json:"workflow_step_execute_id"`
+		Outputs               *map[string]string `json:"outputs"`
```

Comment:
Please use `map[string]string` instead of `*map[string]string`.

Same as https://github.com/slack-go/slack/pull/1060#discussion_r896800825

Reply:
fixed https://github.com/slack-go/slack/pull/1060/commits/9013791a078cb1d461d50e6646bacfe1978c9c81